tools/http_server.py

```diff
@@ -194,8 +194,9 @@ def do_GET(self):
     def do_POST(self):
         # Simple echo server for request reflection
         if "echo_server" in self.path:
+            status = int(self.headers.getheader('x-status', "200"))
```

Comment:
This is working but I don't know python, so if there's a better way please let me know :)

Reply:
that works